### DISCUSSION TITLE
Grad cam for binary classification

### DISCUSSION THREAD

**Discussion by rajanish4** [Project-MONAI/MONAI#5528] [Ideas]
Hi there,

I am currently running Grad cam class for my binary classification model that returns a single value. I believe, the grad cam compute function requires multiple class labels for it to work. 
So, i would like to request a modification of this that can be used for such binary classification tasks.

Thanks

**Comment by wyli** [comment]
thanks, I'm converting this to a feature request https://github.com/Project-MONAI/MONAI/issues/5530
